File: aurea.f90
! Cálculo da razão áurea (phi)
program aurea

        implicit none

        real(kind=8) :: phi = 3
        real(kind=8) :: tmp = -1

        integer(kind=1) :: iter = 0

        do while (abs(phi - tmp) > epsilon(phi))
                tmp = phi
                phi = sqrt(phi + 1)
                iter = iter + 1
        end do

        write (*,'(i2,a,f17.15)') iter,' iterações: ϕ = ',phi

end program aurea

File: aurea.c
// Cálculo da razão áurea (phi)
#include <float.h>
#include <math.h>
#include <stdio.h>
#include <stdlib.h>

int main() {

	double phi = 3;
	double tmp = -1;

	unsigned int iter = 0;

	while (fabs(phi - tmp) > DBL_EPSILON) {
		tmp = phi;
		phi = sqrt(phi + 1);
		iter++;
	}

	printf("%u iterações: ϕ = %1.15f\n",iter,phi);
	return EXIT_SUCCESS;

}
